docs/abi-spec.rst

```diff
@@ -410,15 +410,19 @@ A function description is a JSON object with the fields:
   * ``components``: used for tuple types (more below).
 
 - ``outputs``: an array of objects similar to ``inputs``, can be omitted if function doesn't return anything;
-- ``payable``: ``true`` if function accepts ether, defaults to ``false``;
```

Comment:
@axic One quick thing here… With lists, you can end the bullet points with whatever you want, but keep them consistent, or follow grammatically. So here the semi-colons might not really be needed. It's a very small thing, so feel free to ignore, and I can polish in the future.

Reply:
I kept the original punctuation, not sure why semi-colon was used in the first place.